commands/__tests__/accounts.test.ts

```diff
@@ -37,7 +35,7 @@ describe('commands/accounts', () => {
 
   describe('builder', () => {
     const subcommands = [
-      ['list', { ...list, aliases: 'ls' }],
+      ['list', list],
```

Comment:
I moved this to the list file `command` field to match the pattern we use elsewhere.